charts/external-dns/CHANGELOG.md

```diff
@@ -11,7 +11,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### All Changes
 
-## [v1.12.2] - UNRELEASED
+## [v1.13.0] - 2023-03-30
+
+### All Changes
+
+- Updated _ExternalDNS_ version to [v0.13.5](https://github.com/kubernetes-sigs/external-dns/releases/tag/v0.13.5). ([#3661](https://github.com/kubernetes-sigs/external-dns/pull/3661)) [@GMartinez-Sisti](https://github.com/GMartinez-Sisti)
+- Resolve LB-type Service hostname to create A/AAAA instead of CNAME. ([#3554](https://github.com/kubernetes-sigs/external-dns/pull/3554)) [@cxuu](https://github.com/cxuu)
```

Comment:
The Helm chart change for this PR got reverted.

Reply:
You're right. The revert acted on two PRs. Thanks for the heads up! Fixed.